guides/source/getting_started.md

```diff
@@ -312,8 +312,6 @@ It should look something like the following:
 Rails.application.routes.draw do
   get 'welcome/index'
 
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
```

Comment:
Do we really need to mention everything explicitly? I was just updating the testing guide and found lot of places where we replace the `test the truth` test without explicitly instructing users to replace or delete. I think this change is fine as it is.